modules/apps/portal-security/portal-security-auth-verifier/configs/com.liferay.portal.security.auth.verifier.internal.digest.authentication.configuration.DigestAuthenticationAuthVerifierConfiguration-default.config

```diff
@@ -0,0 +1,3 @@
+enabled=B"true"
```

Comment:
It looks this pull is introducing a bundled configuration file
here.


If that is the case and this pull is fixing a bug please mark this pull as 
*DO NOT BACKPORT*


Please see https://issues.liferay.com/browse/PTR-1770